docs/api/scripts_kg_modules.md

```
@@ -0,0 +1,258 @@
+| Description | *No module description available.* |
+| Args | — |
+| Returns | — |
+
+
+## `scripts\kg\modules\analysis`
+
+**🧠 Docstring Summary**
+
+| Section | Content |
+|---------|---------|
+| Description | Complexity Analysis for Knowledge Graphs
```

markdownlint-cli2 (0.17.2)

Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)